vms/platformvm/block/builder/builder.go

```diff
@@ -102,26 +102,7 @@ func New(
 
 // AddUnverifiedTx verifies a transaction and attempts to add it to the mempool
 func (b *builder) AddUnverifiedTx(tx *txs.Tx) error {
```

Comment:
This function will be removed entirely in the follow-up: https://github.com/ava-labs/avalanchego/pull/2311